manim/mobject/geometry/line.py

```diff
@@ -1015,6 +1015,53 @@ def construct(self):
             return self.angle_value / DEGREES
         return self.angle_value
 
+    @staticmethod
+    def from_three_points(
+        A: np.ndarray, B: np.ndarray, C: np.ndarray, **kwargs
+    ) -> Angle:
+        """The angle between the lines AB and BC.
+
+        This constructs the angle :math:`\angle ABC`.
+
+        Parameters
+        ----------
+        A
+            The endpoint of the first angle leg
+        B
+            The vertex of the angle
+        C
+            The endpoint of the second angle leg
+
+        **kwargs
+            Further keyword arguments are passed to :class:`.Angle`
+
+        Returns
+        -------
+        The Angle calculated from the three points
+
+                    Angle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8),
+                    Angle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, other_angle=True),
+
+        Examples
+        --------
+        .. manim:: AngleFromThreePointsBasicExample
+            :save_last_frame:
+
+            class AngleFromThreePointsBasicExample(Scene):
+                def construct(self):
+                    right_angle = Angle.from_three_points(UP, ORIGIN, RIGHT)
+                    self.add(right_angle)
+
+        .. manim:: AngleFromThreePointsRedExample
+            :save_last_frame:
+
+            class AngleFromThreePointsRedExample(Scene):
+                def construct(self):
+                    right_angle = Angle.from_three_points(LEFT + UP, ORIGIN, RIGHT, stroke_width=8, color=RED)
+                    self.add(right_angle)
```

Comment:
I don't think we need to separate examples, one example with two (or more) angles added is sufficient in my opinion. The color is a good way of holding them apart; the angle is again not a right angle though, so I'd suggest renaming it to `red_angle`, for example.

Reply:
Also, would I use Vgroup like
```scene.add(VGroup(sample_angle, red_angle).arrange(RIGHT)) ```

Reply:
or alternatively,

```
                    sample_angle = Angle.from_three_points(UP, ORIGIN, RIGHT, other_angle=True)
                    red_angle = Angle.from_three_points(LEFT + UP, ORIGIN, RIGHT, stroke_width=8, color=RED)
                    self.add(sample_angle)
                    self.add(red_angle)
```

Reply:
Personally, I would for the sake of simplicity go for
```py
sample_angle = Angle.from_three_points(...)
red_angle = Angle.from_three_points(...)
self.add(sample_angle, red_angle)
```
where the two angles are constructed in a way such that the vertex of the angle is not the same point for both.

Reply:
I just updated the angles and rendered them and they are easily distinguishable from each other!